src/NuGet.Core/NuGet.Commands/ClientCertificatesCommand/ListClientCertRunner.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.Commands
+{
+    public class ListClientCertRunner
+    {
+        public static void Run(ListClientCertArgs args, Func<ILogger> getLogger)
+        {
+            args.Validate();
+            var settings = RunnerHelper.GetSettings(args.Configfile);
+            var clientCertificateProvider = new ClientCertificateProvider(settings);
+
+            var items = clientCertificateProvider.GetClientCertificates();
+
+            if (!items.Any())
+            {
+                getLogger().LogInformation(Strings.NoClientCertificates);
+                return;
+            }
+
+            var clientCertificatesLogs = new List<LogMessage>();
+
+            getLogger().LogInformation(Strings.RegsiteredClientCertificates);
+            getLogger().LogInformation(Environment.NewLine);
+
+            for (var i = 0; i < items.Count; i++)
+            {
+                var item = items[i];
+
+                var builder = new StringBuilder();
+
+                var index = $" {i + 1}.".PadRight(6);
+                var defaultIndentation = string.Empty.PadRight(6);
```

Comment:
```suggestion
                const string defaultIndentation = "      ";
```

This is very subjective. I'd appreciate someone else in my team chiming in. This isn't a performance critical section of code, but I still don't like unnecessary CPU work.